src/parameterization_constants.jl

```diff
@@ -0,0 +1,16 @@
+"""
+Parameters for radiation parameterizations.
+"""
+@with_kw struct ParameterizationConstants{NF<:AbstractFloat}
+    fband::Matrix{Real}
+end    
```

Comment:
without default value skip the `@with_kw` constructor macro. We also should have `fband::Matrix{NF}` for type stability. And I would put everything else that's a constant in our way of thinking about it in here too.